fili-core/src/test/groovy/com/yahoo/bard/webservice/table/PhysicalTableSpec.groovy

```diff
@@ -107,55 +108,52 @@ class PhysicalTableSpec extends Specification {
         table = new ConcretePhysicalTable(
                 name,
                 YEAR.buildZonedTimeGrain(UTC),
-                [] as Set
-                ,
+                [new DimensionColumn(dimension)] as Set,
                 ["dimension": "druidDim"]
         )
 
         then:
-        table.availableIntervalsRef.get() != null
-        table.availableIntervals.isEmpty()
-        table.workingIntervals.isEmpty()
+        table.getAvailability() != null
+        table.getAvailability().isEmpty()
 
         when:
         table.resetColumns(noMetricMetadata, dimensionDictionary)
 
         then:
         table.availableIntervals.containsKey(dimensionColumn)
-        table.availableIntervals.get(dimensionColumn) == intervalSet3
+        table.availableIntervals.get(dimensionColumn).toList() == new SimplifiedIntervalList(intervalSet3) as List
```

Comment:
It seems like we added the simplify interval functionality earlier so the tests will need to check the intervals in simplified version